packages/collections/_test.pony

```diff
@@ -752,3 +753,158 @@ class iso _TestHeap is UnitTest
       t.assert_false(P(h._apply(b)?, h._apply(i)?))
       _verify[P](t, h, b)?
     end
+
+class iso _TestFlags is UnitTest
+  fun name(): String => "collections/Flags"
+
+  fun apply(h: TestHelper) =>
+    // t is for temporary.
+    var t: _TestFlagsFlags = _TestFlagsFlags
+
+    /* As we want to test all binary combinations, we need
+     * four flags to test the following combinations:
+     * 00 01 10 11
+     *
+     * Keeping these as let/val to ensure we don't accidentally
+     * overwrite or mutate them                                  */
+    let set0011: _TestFlagsFlags = _TestFlagsFlags
+                                   .>set(_TestFlagB)
+                                   .>set(_TestFlagA) // 0b0011
```

Comment:
I dont think you need any of these `//` comments. I dont think they add anything. You are doing the comparison immediately after.

Reply:
Agreed - they were there for my own reference before the verification.